src/main/java/Executer.java

```diff
@@ -0,0 +1,36 @@
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+public final class Executer {
```

Comment:
I recommend add convension checker same as Yahya and goda
I did the same in sample calculator